advanced/am-pattern-1/requirements.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Chathuru please adjust according to the previously mentioned license header setting rule.

Reply:
Fix the versioning on https://github.com/Chathuru/kubernetes-apim/commit/1d4d91b99069f609dea7e266cce59f9fa431b59d commit